python/PyQt6/gui/auto_generated/maptools/qgsmaptoolcapture.sip.in

```diff
@@ -331,34 +329,10 @@ Stop capturing
 %End
 
   private:
-    virtual void geometryCaptured( const QgsGeometry &geometry );
```

Comment:
What's happened here? I think we need to make this protected, as otherwise we're breaking api